From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Subject: [PATCH] t/libpthread_depends

libpthread needs to be built before modules which need it.

Signed-off-by: Samuel Thibault <samuel.thibault@ens-lyon.org>

---
 nscd/Depend   |    1 +
 resolv/Depend |    1 +
 rt/Depend     |    1 +
 3 files changed, 3 insertions(+)

--- a/nscd/Depend
+++ b/nscd/Depend
@@ -1 +1,2 @@
 linuxthreads
+libpthread
--- a/resolv/Depend
+++ b/resolv/Depend
@@ -1,2 +1,3 @@
 linuxthreads
 nptl
+libpthread
--- a/rt/Depend
+++ b/rt/Depend
@@ -1,2 +1,3 @@
 linuxthreads
 nptl
+libpthread
